src/search-insights.ts

```diff
@@ -93,7 +93,10 @@ export function activate(context: sourcegraph.ExtensionContext): void {
 
                     const directoryPageProvider = sourcegraph.app.registerViewProvider(`${id}.directory`, {
                         where: 'directory',
-                        provideView,
+                        provideView: async ({ viewer}) => {
```

Comment:
Strange formatting 😬 
```suggestion
                        provideView: async ({ viewer }) => {
```